src/hub/vendor/customer.py

```diff
@@ -36,6 +39,52 @@ def run(self) -> None:
         self.send_to_routes(routes, json.dumps(data))
 
 
+class StripeCustomerDeleted(AbstractStripeHubEvent):
+    def run(self) -> None:
+        logger.info("customer deleted", payload=self.payload)
+        customer_id = self.payload.data.object.id
+        uid = self.payload.data.object.metadata.userid
+        deleted_user = g.subhub_deleted_users.get_user(uid, customer_id)
+        result = {}
+        for d in deleted_user.subscription_info:
+            for k in d.keys():
+                result[k] = result.get(k, 0) + d[k]
+        plan_amount = result.get("plan_amount", 0)
+        nicknames = " ".join(
+            [x.get("nickname") for x in deleted_user.subscription_info]
```

Comment:
Does this refer to the Plan.nickname property? If so, this should be updated to use the `format_plan_nickname` method from the `shared.utils` file.

Reply:
https://github.com/mozilla/subhub/issues/311